docs/requirements.txt

```diff
@@ -1,18 +1,23 @@
-Jinja2==3.1.5
-Markdown==3.3.7
-MarkupSafe==2.1.2
-mike==1.1.2
-mkdocs>=1.5
-mkdocs-awesome-pages-plugin==2.8.0
-mkdocs-mermaid2-plugin==1.1.1
-mkdocs-macros-plugin==0.7.0
-mkdocs-material>=9.5.34
-mkdocs-material-extensions>=1.3
-mkdocs-static-i18n==0.53
-markdown-captions==2.1.2
-Pygments>=2.16
-pymdown-extensions>=10.2
-mkdocs-open-in-new-tab==1.0.2
-mkdocs-include-markdown-plugin==4.0.4
-# ../../mkdocs-include-markdown-plugin
-mkdocs-include-markdown-plugin @ git+https://github.com/clubanderson/mkdocs-include-markdown-plugin@879ff41
+# Core Dependencies
+mkdocs>=1.5,<2.0
+Jinja2>=3.1.5,<4.0
+Markdown>=3.3.7,<4.0
+MarkupSafe>=2.1.2,<3.0
+
+# MkDocs Plugins
+mkdocs-material>=9.5.34,<10.0
+mkdocs-material-extensions>=1.3,<2.0
+mkdocs-awesome-pages-plugin>=2.8.0,<3.0
+mkdocs-mermaid2-plugin>=1.1.1,<2.0
+mkdocs-macros-plugin>=0.7.0,<1.0
```

Comment:
I might have written `<2.0` in this case, since major version number zero is so special.